rust/daemon/main.rs

```diff
@@ -82,4 +83,13 @@ fn main () {
       Err(e) => Err(make_error(&e.to_string()))
     }
   } } );
+
+  let server = ServerBuilder::new(io)
+    .threads(2)
+    .rest_api(RestApi::Unsecure)
+    .cors(DomainsValidation::AllowOnly(vec![AccessControlAllowOrigin::Any]))
+    .start_http(&"0.0.0.0:3030".parse().unwrap()) // Any ip
```

Comment:
Same idea here, what's the convention concerning comments?

Reply:
you'd prefer on top?

Reply:
No no, capital letter + dot, or not. ^^

Reply:
```suggestion
    .start_http(&"0.0.0.0:3030".parse().unwrap()) // Any ip.
```